ynput/ayon-core
# Applications: Live attributes update

## Changelog Description
Update attribute values on settings save.

## Additional info
This helps to update the attribute live on settings change. It does work from this version up. The update does not require restart of server, but the live attributes update is available from server version 1.0.7 .

## Testing notes:
1. Make sure you have server 1.0.7 or newer.
2. Create packages on server addons from this PR.
3. Upload applications addon.
4. Use the version in production/staging bundle.
5. Add new vartiant of applications and save settings.
6. Go to any project anatomy and check if the variant is available in the attribute.
